Template.php

```diff
@@ -1713,6 +1809,13 @@ protected function use_unnamed_params() {
             $dat = trim(str_replace("\n$ris_line", "", "\n$dat"));
           }
         }
+        if (isset($start_page)) { // Sometimes people put end page before start page in reference
+          if (isset($end_page)) {
+             $this->add_if_new("pages", $start_page . "-" . $end_page);
```

Comment:
It looks in this editor like a hyphen between start and end page – should be an en-dash.